src/benchmarklib/external_dbgen_utils.cpp

```diff
@@ -28,8 +28,8 @@ void generate_csv_tables_with_external_dbgen(const std::string& dbgen_path, cons
       auto cmd = std::stringstream{};
       // `2>` in a string seems to break Sublime Text's formatter, so it's split into two strings.
       cmd << "cd " << tables_path << " && " << dbgen_path << "/dbgen -f -s " << scale_factor << " "
-          << additional_cli_args << " "
-          << " -b " << dbgen_path << "/dists.dss";
+          << additional_cli_args << "  -b " << dbgen_path << "/dists.dss >/dev/null 2"
+          << ">/dev/null";
```

Comment:
linebreak looks strange. You are redirecting both stdout and stderr?

Reply:
I did not change anything here from before. Since I did not encounter the problem in the comment, I gust changed it to `<< "/dists.dss >/dev/null 2>&1"`.